cookbooks/aws-parallelcluster-config/metadata.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+name 'aws-parallelcluster-config'
+maintainer 'Amazon Web Services'
+maintainer_email ''
+license 'Apache-2.0'
+description 'Configures AWS ParallelCluster'
+long_description 'Configures AWS ParallelCluster'
+issues_url 'https://github.com/aws/aws-parallelcluster-cookbook/issues'
+source_url 'https://github.com/aws/aws-parallelcluster-cookbook'
+chef_version '17.2.29'
+version '3.0.0'
+
+supports 'amazon', '>= 2'
+supports 'centos', '>= 7'
+supports 'ubuntu', '>= 18.04'
+
+depends 'apt', '~> 7.4.0'
```

Comment:
minor - are all these dependencies needed or now that install and config are split this would be a subset?

Reply:
It could be converted to a subset.